src/dom/DomEvent.js

```diff
@@ -172,15 +172,19 @@ L.DomEvent = {
 	},
 
 	getMousePosition: function (e, container) {
+		var evt = e;
+		if (evt.touches) {
+			evt = evt.touches[0] || evt.changedTouches[0];

```

Comment:
I don't think this is a good idea. After all, the function is called get**Mouse**Position and not getEventPosition.

I presume there will be edge cases where the first touch originates outside the map container, then a second one inside, but `getMousePosition` uses only the first one.

I would argue that a new getTouchPosition**s** function is needed, to translate all the touch points into map coordinates.

Until then, it should be fine to expose just `ev.originalEvents` (as with `keypress` events, see `L.Map._fireDomEvent()`) and let the upper functionality decide which touches (if any) should get more attention and if whether to do some `containerPointToLatLng` calls or whatever.